rocketpy/motors/tank_geometry.py

```diff
@@ -369,28 +369,37 @@ def __init__(self, radius, height, spherical_caps=False, geometry_dict=dict()):
         height : float
             Height of the cylindrical tank, in meters.
         spherical_caps : bool, optional
-            If True, the tank will have spherical caps. The default is False.
+            If True, the tank will have spherical caps. The height of the
+            tank is maintained. The default is False.
```

Comment:
I would expand this documentation (and of the height parameter) to be clearer on what is being considered as height when the spherical caps is either True of False.

Of course this PR already helps on this parameter being more consistent and intuitive, but clear docs will make sure that no confusions are made.